### PR TITLE
openmp support

### PR DESCRIPTION
**Pull request by GiovanniBussi**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Added openmp support

### CODE REVIEW

**Comment by GiovanniBussi** [comment]
@conda-forge-admin, please rerender

**Comment by GiovanniBussi** [comment]
@conda-forge/core I would like to merge this but I received an error from conda-forge-linter.

How should I proceed?

Thanks

Giovanni

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,6 +31,8 @@ requirements:
     - liblapack
     - xdrfile
     - zlib
+  run:
+    - llvm-openmp  # [osx]
```
Comment (isuruf) [inline recipe/meta.yaml]: You need this in host too.

**Comment by GiovanniBussi** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (GiovanniBussi) [reply recipe/meta.yaml]: @isuruf thanks! It looks it is not sufficient however..

**Comment by GiovanniBussi** [comment]
Sorry I misunderstood the message... "I tried to re-render for you, but it looks like there was nothing to do" means that there is no change. I will merge this.

Thanks @isuruf for finding the mistake in openmp.

**Pull request merged**